notebooks/261-fast-segment-anything/261-fast-segment-anything.ipynb

```diff
@@ -18,6 +18,8 @@
     "- [Original Pipeline Inference](#FastSAM-in-Ultralytics)\n",
```

Comment:
<p>Usually, the quantization is made optional by using (1) a widget to enable it and (2) <code>skip</code> extension from <code>utils/skip_kernel_extension.py</code> to control cell execution based on the widget value. Please refer for an example to PR #1411 .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>integrated the extention, thank you!</p>